Разметка всех страниц

RunakK opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/1941383/tasks/5)

RunakK committed:
Добавляет разметку главной страницы

RunakK committed:
добавил файл catalog.html

RunakK committed:
Добавляет разметку страницы каталога

RunakK committed:
закрывает тег HTML

RunakK committed:
правильно закрыл тег main

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <img class="header-logo" src="#" alt="Логотип «Седона»">
```
Иконку логотипа можно сделать ссылкой на главную страницу.

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
+          <li class="navigation-item">
+            <a href="#" class="navigation-link">Главная</a>
+          </li>
+          <li class="navigation-item">
```
для текущей ссылки нужен дополнительный класс, чтобы выделить текущую ссылку от остальных

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
+        <h2 class="visually-hidden">Преимущества Седоны</h2>
+        <article class="advantages-top">
+          <h2 class="advantages-title-top">Седона — небольшой городок в Аризоне,
+            заслуживающий большего!</h2>
```
Если это попытка разбить заголовок на две строки как в макете, то так не делается. Сейчас пока можно в разметке фразу делать одной строкой.

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
+
+      <section class="advantages">
+        <h2 class="visually-hidden">Преимущества Седоны</h2>
+        <article class="advantages-top">
```
article это все-таки что-то типа отдельной статьи, заметки. Я не рекомендую пару строк оборачивать в article. Можно article убрать

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
+            заслуживающий большего!</h2>
+          <p class="advantages-description-top">Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+        </article>
+        <ul class="advantages-list">
+          <li class="advantages-item">
+            <h3 class="advantages-title">Настоящий городок</h3>
+            <p class="advantages-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+          </li>
+          <li class="advantages-item">
+            <img src="#" class="advantages-city" alt="Настоящий городок">
+          </li>
+          <li class="advantages-item">
```
Отлично, что ты сделал преимущества списком. Только отдельный item для картинки избыточен. Лучше добавить img в тот item к которому картинка относится.

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
+              туристов</h3>
+            <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
+          </li>
+          <li class="advantages-item">
+            <img src="#" class="advantages-bridge" alt="Мост дьявола">
+          </li>
```
здесь как и выше, картинку лучше разместить в item к которому она относится

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
+    </footer>
+
+
+    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a
+        href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
```
это можно удалить

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
+    <footer class="page-footer">
+      <section class="footer-contacts">
+        <h2 class="visually-hidden">Давайте дружить</h2>
+        <ul class="contacts-social-list">
+          <li class="footer-social-item"><a href="https://vk.com/htmlacademy" class="social-button"><span
+                class="visually-hidden">Вконтакте</span></a></li>
+          <li class="footer-social-item"><a href="https://t.me/htmlacademy" class="social-button"><span
+                class="visually-hidden">Телеграм</span></a>
```
Здесь все хорошо, только стиль записи поправить бы. Не нужно разрывать тег на две строки, сложно читается. Лучше полностью тег начать с новой строки

vindimm commented on index.html:
```diff
@@ -1,12 +1,152 @@
+      <section class="subscribe">
+        <h2 class="subscribe-title">Подпишитесь на рассылку</h2>
+        <p class="subscribe-description">Только полезная информация и никакого спама, честное бойскаутское!</p>
+        <form action="https://echo.htmlacademy.ru" method="post">
+          <label for="subscribe-email" class="visually-hidden"></label>
+          <input type="email" name="email" id="subscribe-email" class="subscribe-email" placeholder="Ваш e-mail">
+          <button type="submit" class="subscribe-button">ПОДПИСАТЬСЯ</button>
+        </form>
```
Label - это подпись к полю. Она не всегда нужна, в данном случае не вижу смысла в label.
Инпуту с емайлом лучше добавить атрибут обязательного заполнения

vindimm commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Тип жилья</legend>
+            <ul class="catalog-filter-list">
+              <li class="catalog-filter-item"><input class="control-input" type="radio" id="hotel" name="hotel" checked>
+                <label class="control" for="hotel">Гостиница</label>
+              </li>
+              <li class="catalog-filter-item"><input class="control-input" type="radio" id="motel" name="motel">
+                <label class="control" for="motel">Мотель</label>
+              </li>
+              <li class="catalog-filter-item"><input class="control-input" type="radio" id="appartments"
+                  name="appartments">
+                <label class="control" for="appartments">Аппартаменты</label>
+              </li>
+            </ul>
```
Для группы радио баттонов указываем одинаковый name и разные value.
И еще раз - стиль написания. li и input на одной строке здесь плохо читаются.

vindimm commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+          <a href="#" class="button product-card-button">подробнее</a>
+          <button type="submit" class="button product-card-submit">в избранное</button>
+          <img src="#" alt="звёздочка">
+          <button class="button product-button-rating">Рейтинг: 8,5</button>
```
В карточках значение рейтинга это не кнопка, можно сделать просто span

vindimm commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+        <li class="pagination-item">
+          <a href="#" class="pagination-link">4</a>
+        </li>
+        <li class="pagination-item">
+          <a href="#" class="pagination-link">...</a>
+        </li>
```
Троеточие не является интерактивным элементом, нужно убрать href.

vindimm reviewed (changes requested):
В целом все хорошо, проделана значительная работа.

RunakK committed:
Вносит правки

in index.html
-Добавляет дополнительный класс в nav для текущей ссылки

-исправляет перенос строки
-убирает тег article в advantages
-advantages-list  добавляет img внутрь item
-удаляет абзац
-social-list поправляет перенос тегов
-добавляет к email атрибут required

in catalog.html
-catalog-filter поправляет переносы тегов. для группы радиокнопок добавляет name и value
-в рейтинге заменяет тег button на span
-в пагинации убирает href у троеточия
